src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  background-color: rgb(235, 235, 235);
+
+  height: 5000px;
+}
+
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: #fff;
+}
```

Comment:
you don't need all of these, you don't have to scroll your page, width will be 100% anyway, samewith color
```suggestion
```

Reply:
Ok, I remove class Header in CSS file :)